src/timeline/TimelineEventsVisualizer.jsx

```diff
@@ -20,13 +20,12 @@ class TimelineEventsVisualizer extends Visualizer {
         const { conditionSection, tdpSearchSuggestions, highlightedSearchSuggestion, isWide } = this.props;
         const items = this.createItems(conditionSection, tdpSearchSuggestions, highlightedSearchSuggestion);
         const groups = this.createGroupsForItems(this.getMaxGroup(items));
-
+        this.stickySupport = true;
```

Comment:
[DP1] Do we want to have this use some of the utilities defined in Slate for checking what Browser we're using (in `src/lib/slate/constants/environment.js`)? On the one hand that would allow us to actually use regular sticky on other browsers; on the other hand, I don't know we can reasonably expect `window.navigator` to be defined when we launch the app via Smart-on-FHIR. @rdingwell Do you  know anything about how/if the Smart-on-FHIR launcher would scramble any information stored in `window.navigator`?

Reply:
The sticky support is acquiesced from the CSS computed style, since unsupported "position" defaults to "static".  So this solution lets us use regular sticky on other browsers anyway.  Though if we can detect browsers that will probably be a more comprehensive, less hacky solution.

Reply:
Ehhh I think the way we do this now is fine. I retract this request.

Reply:
lol sorry this seecond comment was made before seeing your thoughtful response.  My bad!

Reply:
What about using a util? Something like:
```
// utils/supportsSticky
const supportsSticky = (() => {
  const node = document.createElement('div');
  node.style.position = 'sticky';
  return node.style.position === 'sticky';
})()

export default supportsSticky;```